README.md

```diff
@@ -1,54 +1,63 @@
-[![Documentation](https://codedocs.xyz/xbmc/xbmc.svg)](https://codedocs.xyz/xbmc/xbmc/)
+![Kodi Logo](docs/resources/banner.png)
 
-![Kodi logo](media/banner.png)
-# Kodi Home Theater Software
+[![License](https://img.shields.io/badge/license-GPLv2-blue.svg?style=flat-square)](LICENSE.md)
+[![HitCount](http://hits.dwyl.io/xbmc/xbmc.svg)](http://hits.dwyl.io/xbmc/xbmc)
+[![Documentation](https://img.shields.io/badge/code-documented-brightgreen.svg?style=flat-square)](https://codedocs.xyz/xbmc/xbmc/)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/xbmc/xbmc/pulls)
+[![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat-square)](#how-to-contribute)
+[![Build](https://img.shields.io/badge/CI-jenkins-brightgreen.svg?style=flat-square)](http://jenkins.kodi.tv/)
+[![Commits](https://img.shields.io/github/commits-since/xbmc/xbmc/17.6-Krypton.svg?style=flat-square)](https://github.com/xbmc/xbmc/commits/master)
```

Comment:
`https://img.shields.io/github/commits-since/xbmc/xbmc/latest.svg?style=flat-square`

Reply:
Fixed but I don't know if I'll keep this *shield*. Too slow to load most of the times.